requirements.txt

```diff
@@ -3,5 +3,5 @@ hydra-genetics==0.15.0
 jinja2==3.0.1
 pandas>=1.3.1
 singularity==3.0.0
-snakemake==7.8.0
+snakemake>=7.8.3
```

Comment:
I would prefer to have fixed version or 
snakemake>=7.8.3,<8
so that we do not allow braling changes

Reply:
In my experience, freezing versions usually cause more issues than they solve, but making sure that it doesn't accept a new major version I can definitely see the point of.